bgpd/bgp_route.c

```diff
@@ -10179,18 +10181,14 @@ void route_vty_out_detail(struct vty *vty, struct bgp *bgp,
 		}
 	}
 
-	const struct prefix *p = bgp_dest_get_prefix(bn);
-	if (p->family == AF_INET || p->family == AF_INET6)
-		rpki_validation_state = hook_call(bgp_rpki_prefix_status,
-						  path->peer, path->attr, p);
-	if (rpki_validation_state) {
+	if (curr_state != RPKI_NOT_BEING_USED) {
```

Comment:
IMO, prefixed with rpki_ is easier to read, like before.

Reply:
fixed